doc/02-Installation.md

```diff
@@ -1,145 +1,213 @@
+
+```bash
+rpm --import https://packages.icinga.com/icinga.key
+wget https://packages.icinga.com/subscription/rhel/ICINGA-release.repo -O /etc/yum.repos.d/ICINGA-release.repo
+```
+<!-- {% endif %} -->
+
+<!-- {% if sles %} -->
+<!-- {% if not icingaDocs %} -->
+### SLES Repository <a id="sles-repository"></a>
+<!-- {% endif %} -->
+!!! info
+
+    A paid repository subscription is required for SLES repositories. Get more information on
+    [icinga.com/subscription](https://icinga.com/subscription)
+
+    Don't forget to fill in the username and password section with appropriate credentials in the local .repo file.
 
-Ubuntu:
+```bash
+rpm --import https://packages.icinga.com/icinga.key
 
+zypper ar https://packages.icinga.com/subscription/sles/ICINGA-release.repo
+zypper ref
 ```
+<!-- {% endif %} -->
+
+<!-- {% if ubuntu %} -->
+<!-- {% if not icingaDocs %} -->
+### Ubuntu Repository <a id="ubuntu-repository"></a>
+<!-- {% endif %} -->
+
+```bash
 apt-get update
 apt-get -y install apt-transport-https wget gnupg
 
 wget -O - https://packages.icinga.com/icinga.key | apt-key add -
 
 . /etc/os-release; if [ ! -z ${UBUNTU_CODENAME+x} ]; then DIST="${UBUNTU_CODENAME}"; else DIST="$(lsb_release -c| awk '{print $2}')"; fi; \
- echo "deb https://packages.icinga.com/ubuntu icinga-${DIST}-testing main" > \
- /etc/apt/sources.list.d/${DIST}-icinga-testing.list
- echo "deb-src https://packages.icinga.com/ubuntu icinga-${DIST}-testing main" >> \
- /etc/apt/sources.list.d/${DIST}-icinga-testing.list
+ echo "deb https://packages.icinga.com/ubuntu icinga-${DIST} main" > \
+ /etc/apt/sources.list.d/${DIST}-icinga.list
+ echo "deb-src https://packages.icinga.com/ubuntu icinga-${DIST} main" >> \
+ /etc/apt/sources.list.d/${DIST}-icinga.list
 
 apt-get update
 ```
+<!-- {% endif %} -->
 
-### Installing Icinga DB <a id="installing-icingadb"></a>
+## Installing Icinga DB Package <a id="installing-icinga-db-package"></a>
 
-RHEL/CentOS 8/Fedora:
+Use your distribution's package manager to install the `icingadb` package as follows:
 
-```
-dnf install icingadb
-systemctl enable icingadb
-systemctl start icingadb
-```
-
-RHEL/CentOS 7:
-
-```
+<!-- {% if amazon_linux %} -->
+<!-- {% if not icingaDocs %} -->
+#### Amazon Linux 2
+<!-- {% endif %} -->
+```bash
 yum install icingadb
-systemctl enable icingadb
-systemctl start icingadb
 ```
+<!-- {% endif %} -->
 
-SUSE:
-
-```
-zypper install icingadb
-```
+<!-- {% if centos %} -->
+<!-- {% if not icingaDocs %} -->
+#### CentOS
+<!-- {% endif %} -->
+!!! info
 
-Debian/Ubuntu:
+    Note that installing Icinga DB is only supported on CentOS 7 as CentOS 8 is EOL.
 
+```bash
+yum install icingadb
 ```
+<!-- {% endif %} -->
+
+<!-- {% if debian or ubuntu %} -->
+<!-- {% if not icingaDocs %} -->
+#### Debian / Ubuntu
+<!-- {% endif %} -->
+```bash
 apt-get install icingadb
 ```
+<!-- {% endif %} -->
 
-### Installing Icinga DB Redis <a id="configuring-icingadb-mysql"></a>
+<!-- {% if rhel %} -->
+#### RHEL 8
```

Comment:
> I think we should just make this future-proof by adding "or later" to the heading. RHEL 9 is out already, we don't have packages yet, but otherwise we'd have to update this again soon.

Make sense!